scene/resources/texture.cpp

```diff
@@ -1762,7 +1762,7 @@ void GradientTexture::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("_update"), &GradientTexture::_update);
 
 	ADD_PROPERTY(PropertyInfo(Variant::OBJECT, "gradient", PROPERTY_HINT_RESOURCE_TYPE, "Gradient"), "set_gradient", "get_gradient");
-	ADD_PROPERTY(PropertyInfo(Variant::INT, "width", PROPERTY_HINT_RANGE, "1,2048,1,or_greater"), "set_width", "get_width");
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "width", PROPERTY_HINT_RANGE, "32,4096"), "set_width", "get_width");
```

Comment:
Shouldn't there be a min value?

Reply:
Ah nevermind, I guess the step is optional.
I don't think we should restrict min value to 32, it can be an artistic choice to use a gradient with e.g. 4 steps.

Reply:
Changed to allow widths as low as 1. I also applied the same change to CurveTexture and Curve3Texture for consistency.